phase/configure_k0s.go

```diff
@@ -51,7 +51,7 @@ func (p *ConfigureK0s) Run() error {
 
 func (p *ConfigureK0s) validateConfig(h *cluster.Host) error {
 	log.Infof("%s: validating configuration", h)
-	output, err := h.ExecOutput(h.Configurer.K0sCmdf(`validate config -c "%s"`, h.K0sConfigPath()))
+	output, err := h.ExecOutput(h.Configurer.K0sCmdf(`validate config --config "%s"`, h.K0sConfigPath()))
```

Comment:
I think this will break in future upgrades if the config syntax changes as its done using the old binary 🤔

Reply:
Yup. We might need to upload/download the new version first to some temp file and use it to validate the config. I'd leave it for follow-up PRs though